test/common/test_runner.go

```diff
@@ -77,6 +74,8 @@ func Setup(t *testing.T, runInParallel bool) *Client {
 	// Create a new namespace to run this test case.
 	baseFuncName := helpers.GetBaseFuncName(t.Name())
 	namespace := helpers.MakeK8sNamePrefix(baseFuncName)
+	// Append random postfix for the namespace name.
+	namespace = helpers.AppendRandomString(namespace)
```

Comment:
Could we make this behavior optional? Is there some way to find all namespaces with random suffix and clean them from tests?